### DISCUSSION TITLE
2023-10-10

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#630] [daily_meeting]
### 오늘의 할 일
- 프론트엔드
  - 미션 마무리
- 백엔드
  - 부하테스트 논의
  - 개별 태스크 진행

### 개인 일정
- 노아 (3-4 근로)

### 나중에 할 일
- 폴링 구현 하루 이틀 정도 더 소요 예정
- 회원 탈퇴(폴링, 배포 이후에)

**Comment by aak2075** [comment]
### 체크인

10점 - 목이 조금 아프고 어지럽네요

### 체크아웃

10점 - 돈받고 테스트 짜줄 사람 구합니다

**Comment by jaehee329** [comment]
### 체크인
5점 - 콧물이..나와요

### 체크아웃
4점 - 콧물이...나와요

**Comment by yeopto** [comment]
### 체크인

7점 - 몸살 기운이 아직 남아있어서 컨디션이 좋지 않네요

### 체크아웃

6점 - Next.js 너무 어렵다. 몸도 안좋다.

**Comment by woo-jk** [comment]
### 체크인
4점 - 잠을 너무 조금 잤어요

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 컨디션이 조금 난조긴 하지만 아침에 공부하니까 기분이 좋아졌어요

### 체크아웃
10점 - 이제 진짜 끝이 보이네요

**Comment by woosung1223** [comment]
### 체크인
9점 - 화요일이지만 나쁘지 않네요

### 체크아웃
8점 - 코테를 본다니 너무 슬퍼요

**Comment by nlom0218** [comment]
### 체크인

5점 - 아침에 왼쪽 갈비쪽을 삐긋해서 불편하네욤.. 😭

### 체크아웃

8점 - 알듯말듯 NextJS
